utils/src/main/kotlin/Extensions.kt

```diff
@@ -244,9 +244,9 @@ fun JsonNode?.fieldsOrEmpty(): Iterator<Map.Entry<String, JsonNode>> = this?.fie
 fun JsonNode?.textValueOrEmpty(): String = this?.textValue().orEmpty()
 
 /**
- * Converts this [Long] from bytes to mebibytes (MiB).
+ * Converts this [Number] from bytes to mebibytes (MiB).
  */
-fun Long.bytesToMiB(): Double = this / (1024.0 * 1024.0)
+fun Number.bytesToMiB(): Double = toDouble() / (1024 * 1024)
```

Comment:
How about also moving this down to keep the alphabetic order of the extension functions?

Reply:
Forgot about this, done.